types/node/index.d.ts

```diff
@@ -87,6 +87,12 @@ interface SharedArrayBuffer {
     slice(begin?: number, end?: number): SharedArrayBuffer;
 }
 
+interface PromiseSettlement<T> {
```

Comment:
This is also in https://github.com/microsoft/TypeScript/pull/34065.

And it doesn’t appear to be used in this PR anywhere else.